martin/src/srv/server.rs

```diff
@@ -115,7 +115,8 @@ pub fn new_server(config: SrvConfig, state: ServerState) -> MartinResult<(Server
 
         let app = App::new()
             .app_data(Data::new(state.tiles.clone()))
-            .app_data(Data::new(state.cache.clone()));
+            .app_data(Data::new(state.cache.clone()))
+            .app_data(Data::new(state.preferred_encoding));
```

Comment:
I think we should store `SrvConfig` object in the data.  Read the steps in https://github.com/maplibre/martin/issues/1185 -- here you have almost the same thing - a configuration value you can use in `get_tile` in `tiles.rs`